src/contexts/InputContext.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { createContext, useContext, useState } from 'react'
+
+const InputFormContext = createContext()
+
+const InputFormProvider = ({ children }) => {
+    const [notes, setNotes] = useState([])
+
+    return (
+        <InputFormContext.Provider value={{ notes, setNotes }}>
```

Comment:
Suggestion : Don't pass the setter function directly. 

Make a custom function for your setter and then pass that function.